src/js/models/earth.js

```diff
@@ -20,8 +20,9 @@ export default class Earth {
                 map:    earthTexture,
             });
             let earthMesh = new THREE.Mesh(earthGeo, earthMaterial);
-            earthMesh.rotation.x += 0.5;
-            earthMesh.position.x += 5;
+            earthMesh.scale.set(5,5,5);
```

Comment:
Doing the same for earth size could help too. Have it be a constant in index.js and then pass it into the constructor